### PR TITLE
docs(arc42):[TRACEFOSS-915] Introduced documentation for quality requ…

### PR DESCRIPTION
**Pull request by mkanal**
…irements chapter

### CODE REVIEW

File: docs/src/uml-diagrams/arc42/quality/quality-tree.puml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@startmindmap
```
[the remark this answers is not in the document]
Comment (tbraun-zf) [reply docs/src/uml-diagrams/arc42/quality/quality-tree.puml:1]: Do I misunderstand this? The (qaulity-tree.)puml file is in uml-diagram folder and in the adoc file its only included but not directly in the code?

File: docs/src/docs/arc42/quality/quality-scenarios.adoc [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+|ID |Scenario
+
+|M01
+|A developer with basic knowledge of the Traceability Use Case looks for an introduction to the Traceability architecture. He or she gets the idea of essential design very fast.
+
+|M02
+|A senior developer looks for a reference implementation of the Traceability Use case functionalities. He or she gets it within the source code.
+
+|M03
+|A developer wants to implement new features. He is able to add it to the source code easily.
```
[the remark this answers is not in the document]
Comment (tbraun-zf) [reply docs/src/docs/arc42/quality/quality-scenarios.adoc]: Done with new commit.

**Review by mkanal — COMMENTED** [review]
I would recommend to put the image content under docs/src/images/user-guide because user-manual is an own document and not part of the ARC42 documentation, (docs/src/images/arc42/user-guide)

**Review by mkanal — COMMENTED** [review]
Thank you very much.

**Pull request merged**